ansible/ansible.cfg

```diff
@@ -30,6 +30,9 @@ ansible_python_interpreter = /opt/local/bin/python
 [hosts:freebsd]
 ansible_python_interpreter = /usr/local/bin/python
 
+[hosts:fedora30]
```

Comment:
this will become `hosts:fedora` on the next update, from now on we need python3, there is no python2-dnf from Fedora 30+ which makes Ansible bork as per https://github.com/ansible/ansible/issues/54855